packages/shared-pure/src/types/api/DetailedTvlApi.ts

```diff
@@ -44,9 +46,19 @@ export const DetailedTvlApiToken = z.object({
   assetId: branded(z.string(), AssetId),
   tvl: z.number(),
 })
+
+export const DetailedTvlApiTopToken = z.object({
+  assetId: branded(z.string(), AssetId),
+  chainId: branded(z.number(), ChainId),
+  valueType: branded(z.string(), ValueType),
+})
+
+export type DetailedTvlApiTopToken = z.infer<typeof DetailedTvlApiTopToken>
+
 export type DetailedTvlApiToken = z.infer<typeof DetailedTvlApiToken>
 
 export const DetailedTvlApiProject = z.object({
+  topTokens: z.array(DetailedTvlApiTopToken),
```

Comment:
let's do Top Tokens per value type, I know right now it is not an issue, but in the future can be helpful (when we have more NMV and EMV tokens in config)